packages/manager/apps/pci-kubernetes/src/components/service/ClusterNetwork.component.tsx

```diff
@@ -39,7 +41,14 @@ export default function ClusterNetwork({
     kubeDetail.privateNetworkId,
   );
 
-  const shouldLoadSubnets = !!kubeDetail.privateNetworkId;
+  const {
+    data: regionInformations,
+    isPending: isPendingRegionInformation,
+  } = useRegionInformations(projectId, kubeDetail.region);
+
+  const shouldLoadSubnets =
+    !!kubeDetail.privateNetworkId &&
+    isMonoDeploymentZone(regionInformations?.type);
```

Comment:
The PR name says it's to disable 3AZ but here you are only allowing 1AZ, what about LZ deployments ?